otx/algorithms/classification/adapters/deep_object_reid/configs/efficientnet_b0/template_experimental.yaml

```diff
@@ -0,0 +1,57 @@
+# Description.
+model_template_id: Custom_Image_Classification_EfficinetNet-B0
+name: EfficientNet-B0
+task_type: CLASSIFICATION
+task_family: VISION
+instantiation: "CLASS"
+summary: Provides better performance on large datasets, but may be not so stable in case of small amount of training data.
+application: ~
+
+# Algo backend.
+framework: OTEClassification v1.2.3
+
+# Task implementations.
+entrypoints:
+  base: torchreid_tasks.train_task.OTXClassificationTrainingTask
```

Comment:
is it `torchreid_tasks`? I know that `otx` cannot find `torchreid_task`.

Reply:
Yeah. since dor does not do train/inference task, this will not be run. after testing i remove this config, and if incurs no errors, i will remove it.